### PR TITLE
RSLC-4 As a user I should only be able to create a maximum of 3 targets

### PR DESCRIPTION
**Pull request by diegoxanderdx**
#### Board:
* [RSLC-4](https://trello.com/c/Q0VR7Ojf)
---
#### Description:
As a user I should only be able to create a maximum of 3 targets
---
#### Notes:
Please ckeck this commits
[add max amount of targets per user](https://github.com/diegoxanderdx/Target/pull/6/commits/9e4320a17ae3ced8067b097092128c0284353993)
[rubocop corrections](https://github.com/diegoxanderdx/Target/pull/6/commits/f0e4f52d32e405bc69a1e9ae8c278c16308abb4c)
---
#### Tasks:
- [x] Create validation in the model
- [x] Create error message
- [x] Edit spec for targets controller
---
#### Risk:
*
---

### CODE REVIEW

File: app/models/target.rb [hunk 1]
```diff
@@ -24,4 +24,13 @@ class Target < ApplicationRecord
   validates :title, :radius, :latitude, :longitude, presence: true
   validates :radius, numericality: { greater_than: 0 }
   validates :latitude, :longitude, numericality: true
+  validate :user_targets_count, unless: -> { user.nil? }, on: :create
+
+  private
+
+  def user_targets_count
+    return if user.targets.count < 3
```
Comment (enzofab91) [inline app/models/target.rb:32]: This 3 can be moved to a constant, wdyt?

File: config/locales/en.yml [hunk 2]
```diff
@@ -10,3 +10,7 @@ en:
       invalid_content_type: Invalid content type header
       missing_param: A required param is missing
       not_found: Couldn't find the record
+  model:
+    target:
+      errors:
+        invalid_amount: There are already 3 targets
```
Comment (brunoparma88) [inline config/locales/en.yml:16]: max_targets: Already has the maximum target

it is better to put a representative name to the variable. And also a generic description.

File: app/models/target.rb (hunk 1 above)
Comment (diegoxanderdx) [reply app/models/target.rb]: Great!

File: config/locales/en.yml (hunk 2 above)
Comment (diegoxanderdx) [reply config/locales/en.yml]: great it's more legible

**Pull request merged**